src/eventBus.js

```diff
@@ -0,0 +1,17 @@
+import { reactive } from 'vue'
+
+const eventBus = reactive({
+  events: {},
+  emit(event, data) {
+    if (!this.events[event]) return
```

Comment:
I know this might be a small thing, but would you be okay with us using curly braces on this if/return, to maintain more clarity in this bit of code?
```suggestion
    if (!this.events[event]) {
      return
    }
```